spec/rzo_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+RSpec.describe Rzo do
+  it 'has a version number' do
+    expect(Rzo::VERSION).not_to be nil
+  end
+
+  it 'responds to .version' do
+    expect(described_class).to respond_to :version
+  end
+
+  it '.version == Rizzo::VERSION' do
+    expect(described_class.version).to eq(Rzo::VERSION)
+  end
+
+  xit 'does something useful' do
```

Comment:
we could have a couple `.rizzo.json` files and see that the generated Vagrantfile matches a fixture.